recipes/insurveyor/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "insurveyor" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/kensung-lab/INSurVeyor/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 33c85157892d3256abc96bb2a9053f05da9dcf55befeed720e965577db0b78b5
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - htslib ==1.13
```

Comment:
Is it really necessary to pin htslib so strictly?

Reply:
The software will run with most recent versions, however the software was extensively tested with 1.13, and the results we will soon publish were produced with that version. I am afraid that if small changes are introduced in future versions of htslib, the results may be inconsistent. Do you see any drawbacks in pinning the release? Will it be deprecated in the future?

Reply:
All other packages are built with 1.16, so it's impossible to install this in the same environment as them.

Reply:
Ok. I will test it with 1.16 and remove the constraint on 1.13 in the next release.